web/test/jobs/app_uninstalled_job_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class AppUninstalledJobTest < ActiveJob::TestCase
+  # test "the truth" do
```

Comment:
Do we maybe want to add a quick test to see that the db rows are in fact nuked? We have fixture support so it should be a fairly low lift to set up.